CMakeLists.txt

```diff
@@ -401,6 +402,7 @@ if(BUILD_EXAMPLES)
 endif(BUILD_EXAMPLES)
 
 if(BUILD_TESTS)
+    enable_testing()
```

Comment:
I think this change isn't needed since it's already written on https://github.com/tiny-dnn/tiny-dnn/blob/a8de26ad40d955908c5ec4fc946c3c67dd381c6c/test/CMakeLists.txt#L2

Reply:
But `enable_testing()` needs to be called in the root CMakeLists.txt:
```
$ git checkout v1.0.0a3
[...]
$ mkdir build; cd build
$ cmake .. -DBUILD_TESTS=ON -DCMAKE_INSTALL_PREFIX=$PWD/install
[...]
$ make
[...]
$ make install
[...]
$ ctest .
Test project /tmp/tiny-dnn/build
No tests were found!!!
$ git cherry-pick b1bef
$ cmake ..
$ ctest .
Test project /tmp/tiny-dnn/build
    Start 1: all_tests
[...]
```

Reply:
> But enable_testing() needs to be called in the root CMakeLists.txt

I didn't know about that. It seems you're correct according to the cmake manual. 

https://cmake.org/cmake/help/latest/command/enable_testing.html
> Therefore, this command should be in the source directory root.

Then how about removing duplicating command in `test/CMakeLists.txt`?
But I checked that isn't the the cause of multiple `run_tests` projects. :(
Need to investigate more...